MASH-FRET/restruct_trace_file.m

```diff
@@ -1,36 +1,82 @@
-function restruct_trace_file(pname,varargin)
+function restruct_trace_file(varargin)
```

Comment:
After restructuring the intensity units are lost (e.g. counts). @mca-sh could you recheck the regex?

Reply:
Thank you, it was due a wrong assignment of the units string. It should be fixed in branch MASH-code-6

Reply:
👍